frontend/pre_analysis.py

```diff
@@ -1,19 +1,35 @@
 from collections import OrderedDict
+from frontend.import_handler import ImportHandler
 import ast
 
 
 class PreAnalyzer:
     """Analyzer for the AST to give some configurations before the type inference"""
```

Comment:
This docstring could say what information it actually collects. "Some configurations" doesn't say a lot.